### PR TITLE
Update playbook docs for Butler management

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR updates our policy for Butler versioning and documents how to set registry permissions for the PP service accounts.

### CODE REVIEW

**Review by hsinfang — APPROVED** [review]

**Comment by hsinfang** [comment]
Looks good to me. Thank you for the updates!

File: doc/playbook.rst [hunk 1]
```diff
@@ -257,6 +263,37 @@ In our case, we want to migrate to the versions that ``/repo/embargo`` is using,
    However, when using ``butler migrate`` to update ``dimensions-config``, you should delete all existing pods to ensure that their replacements have the correct version.
    This can be done using ``kubectl delete pod`` or from Argo CD (see `Development Service`_).
 
+Updating Table Permissions
+--------------------------
+
+Some ``dimensions-config`` migrations add new tables to the Butler registry schema.
+When this happens, our service accounts need to be explicitly given permission to work with those new tables.
+
+To update permissions, use ``psql`` to log in to the registry database as the owner (``pp`` for our dev repo).
```
Comment (erinleighh) [inline doc/playbook.rst:272]: Everything looks good to me as someone who is familiar with accessing `psql`. Can we safely assume everyone using this guide will have that core knowledge? If yes, then this is good to go. If not, I think a little more instruction would be helpful.

**Review by erinleighh — APPROVED** [review]

File: doc/playbook.rst (hunk 1 above)
Comment (kfindeisen) [reply doc/playbook.rst:272]: I guess it should be more self-contained. Would a link to the `Databases` section be enough? I'm not sure exactly what you mean by "accessing" `psql`, given that it's a program.

Comment (erinleighh) [reply doc/playbook.rst:272]: Yes, a link back to the Databases section would be perfect!

**Pull request merged**